packages/storybook/.storybook/preview.js

```diff
@@ -1,10 +1,10 @@
 import '@department-of-veterans-affairs/formation/dist/formation.min.css';
 import './style.scss';
 import '@department-of-veterans-affairs/formation/dist/formation';
-import { withHTML } from '@whitespace/storybook-addon-html/react';
+import {withHTML} from '@whitespace/storybook-addon-html/react';
 
-import 'web-components/dist/component-library/component-library.css';
-import { applyPolyfills, defineCustomElements } from 'web-components/loader';
+import '@department-of-veterans-affairs/component-library/dist/main.css';
+import {applyPolyfills, defineCustomElements} from '@department-of-veterans-affairs/component-library';
```

Comment:
The imports changed now that we're getting from the `core` export.

Reply:
Also a bit of code formatting changes. Might look into how Prettier is set up in this new monorepo.